src/cascadia/TerminalApp/TitlebarControl.idl

```diff
@@ -7,14 +7,18 @@ namespace TerminalApp
     {
         WindowVisualStateNormal = 0,
         WindowVisualStateMaximized,
-        WindowVisualStateIconified
+        WindowVisualStateIconified,
+        WindowVisualStateFocused,

```

Comment:
I'm still pretty worried about including these in the list of window states. I think they *need* to be a separate enum. Here's why:

The window is always aware of its current visual state, right...

What happens if you *maximize* it, then *un-focus it*, and then *focus it again*?

The visual state, according to the window, is no longer "maximized." It's "focused." It totally forgot that it was maximized!

We need to track the focus state separately from whether the window is normal, maximized or iconified.

Reply:
Oh, I thought I had fixed that🤔, nvm, doing it asap